lib/interceptor/cache.js

```diff
@@ -0,0 +1,185 @@
+'use strict'
+
+const util = require('../core/util')
+const CacheHandler = require('../handler/cache-handler')
+const MemoryCacheStore = require('../cache/memory-cache-store')
+const CacheRevalidationHandler = require('../handler/cache-revalidation-handler')
+const { UNSAFE_METHODS } = require('../util/cache.js')
+
+const AGE_HEADER = Buffer.from('age')
```

Comment:
I am not entirely sure why `age` here is lowercased. If it is `Age`, we can re-use `getWellKnownHeaderBuffer`.